RecoEgamma/Configuration/python/RecoEgamma_cff.py

```diff
@@ -48,10 +48,19 @@
 from RecoHI.HiEgammaAlgos.photonIsolationHIProducer_cfi import photonIsolationHIProducerpp
 from RecoHI.HiEgammaAlgos.photonIsolationHIProducer_cfi import photonIsolationHIProducerppGED
 from RecoHI.HiEgammaAlgos.photonIsolationHIProducer_cfi import photonIsolationHIProducerppIsland
+from RecoHI.HiCentralityAlgos.HiCentrality_cfi import hiCentrality
+from RecoHI.HiCentralityAlgos.HiClusterCompatibility_cfi import hiClusterCompatibility
+pp_on_XeXe_2017.toModify(hiCentrality,
```

Comment:
All changes to a module should be done in the same python file that the module is declared in [1]. Please move it inside RecoHI.HiCentralityAlgos.HiCentrality_cfi

[1] https://twiki.cern.ch/twiki/bin/view/CMSPublic/SWGuideCmsDriverEras#Configuring_using_an_era

Reply:
Done in the new commit